app/src/assets/localization/en/run_details.json

```
@@ -5,11 +5,12 @@
   "anticipated": "Anticipated steps",
   "apply_stored_data": "Apply stored data",
   "apply_stored_labware_offset_data": "Apply stored Labware Offset data?",
-  "cancel_run_alert_info": "Doing so will terminate this run, drop any attached tips in the trash container and home your robot.",
+  "cancel_run_alert_info_flex": "Doing so will terminate this run and home your robot.",
+  "cancel_run_alert_info_ot2": "Doing so will terminate this run, drop any attached tips in the trash container and home your robot.",
```

```suggestion
  "cancel_run_alert_info_ot2": "Doing so will terminate this run, drop any attached tips in the trash container, and home your robot.",
```
🤓 we could add a serial comma here but it's not critical